psa-crypto/src/lib.rs

```diff
@@ -43,6 +43,7 @@
 
 #[cfg(feature = "with-mbed-crypto")]
 pub mod operations;
+#[cfg(feature = "with-mbed-crypto")]
```

Comment:
I don't think this `cfg` gate is needed for that module!

Reply:
For `types`? Now `get_key_attributes` is in `Attributes`, it needs access to some `mbed-crypto` specific code. Without that, `ci.sh` gives a load of `not found in` errors

Reply:
Actually, I could make it more local. Adding a gate for `use crate::initialized` and one for the method.